docker-compose.yml

```diff
@@ -0,0 +1,100 @@
+services:
+  zen-mcp:
+    build:
+      context: .
+      dockerfile: Dockerfile
+      target: runtime
+    image: zen-mcp-server:latest
+    container_name: zen-mcp-server
+    
+    # Container labels for traceability
+    labels:
+      - "com.zen-mcp.service=zen-mcp-server"
+      - "com.zen-mcp.version=1.0.0"
+      - "com.zen-mcp.environment=production"
+      - "com.zen-mcp.description=AI-powered Model Context Protocol server"
+    
+    # Environment variables
+    environment:
+      # Default model configuration
+      - DEFAULT_MODEL=${DEFAULT_MODEL:-auto}
+      
+      # API Keys (use Docker secrets in production)
+      - GEMINI_API_KEY=${GEMINI_API_KEY}
+      - GOOGLE_API_KEY=${GOOGLE_API_KEY}
+      - OPENAI_API_KEY=${OPENAI_API_KEY}
+      - ANTHROPIC_API_KEY=${ANTHROPIC_API_KEY}
+      - XAI_API_KEY=${XAI_API_KEY}
+      - DIAL_API_KEY=${DIAL_API_KEY}
+      - DIAL_API_HOST=${DIAL_API_HOST}
+      - DIAL_API_VERSION=${DIAL_API_VERSION}
+      - OPENROUTER_API_KEY=${OPENROUTER_API_KEY}
+      - CUSTOM_API_URL=${CUSTOM_API_URL}
+      - CUSTOM_API_KEY=${CUSTOM_API_KEY}
+      - CUSTOM_MODEL_NAME=${CUSTOM_MODEL_NAME}
+      
+      # Logging configuration
+      - LOG_LEVEL=${LOG_LEVEL:-INFO}
+      - LOG_MAX_SIZE=${LOG_MAX_SIZE:-10MB}
+      - LOG_BACKUP_COUNT=${LOG_BACKUP_COUNT:-5}
+      
+      # Advanced configuration
+      - DEFAULT_THINKING_MODE_THINKDEEP=${DEFAULT_THINKING_MODE_THINKDEEP:-high}
+      - DISABLED_TOOLS=${DISABLED_TOOLS}
+      - MAX_MCP_OUTPUT_TOKENS=${MAX_MCP_OUTPUT_TOKENS}
+      
+      # Server configuration
+      - PYTHONUNBUFFERED=1
+      - PYTHONPATH=/app
+      - TZ=${TZ:-UTC}
+    
+    # Volumes for persistent data
+    volumes:
+      - ./logs:/app/logs
+      - zen-mcp-config:/app/conf
```

Comment:
worth adding:

- /etc/localtime:/etc/localtime:ro

to the volumes as this would then correctly handle dates and time stamps